docs/src/languages/php/_index.md

```diff
@@ -304,16 +210,197 @@ markdownify=false
+| `show_source` | This function shows a syntax highlighted version of a named PHP source file. Rarely useful outside of development. |
+
+## Alternate start commands
+
+For further control over your app or to run a custom PHP script, a web server or a thread-based worker process, set a `start` command.
+See more on [web commands reference](../../create-apps/app-reference.md#web-commands).
+
+PHP-CLI (command line interface) is the mode used for alternate start commands which differs from PHP-CGI (common gateway interface).
+PHP-CGI is the mode used by default in the general runtime.
+If you want to use PHP-CGI with PHP-FPM(FastCGI Process Manager), in the `start` commands, use the `/usr/bin/start-php-app` symlink instead of `php`.
+
+Note that the `start` commands must run in the foreground and are executed before the [deploy hook](../../create-apps/hooks/hooks-comparison.md).
+That means that PHP-FPM can't run simultaneously along with another persistent process such as [ReactPHP](https://github.com/platformsh-examples/platformsh-example-reactphp) or [Amp](https://github.com/platformsh-examples/platformsh-example-amphp).
+If you need both, they have to run in separate containers.
+
+Set up alternate start commands to:
+
+{{< codetabs >}}
+
+---
+title=Run a custom script
+file=none
+highlight=false
+---
+
+1. Add your script in a PHP file,
+2. Specify an alternative `start` command by adapting the variable:
+
+    ```yaml {location=".platform.app.yaml"}
+    web:
+        commands:
+            start: php <PATH_TO_FILE>.php
+    ```
+
+<--->
+
+---
+title=Run a custom web server
+file=none
+highlight=false
+---
+
+1. Add your web server's code in a PHP file,
+2. Specify an alternative `start` command by adapting the variable:
+
+    ```yaml {location=".platform.app.yaml"}
+    web:
+        commands:
+            start: php <PATH_TO_FILE>.php
+        upstream:
+                socket_family: tcp
+                protocol: http
+    ```
+
+    By configuring your app to connect via a TCP socket, the `$PORT` environment variable is set.
+    You might have to configure your app to connect via the `$PORT` TCP socket, especially when using web servers like [Swoole](swoole.md), or [Roadrunner](https://github.com/roadrunner-server/roadrunner).
+3. You might also need to override redirects to let the custom web server handle them.
+
+    ```yaml {location=".platform.app.yaml"}
+    locations:
+            "/":
+                allow: false
+                passthru: true
+    ```
+
+<--->
+
+---
+title=Run specific tasks
+file=none
+highlight=false
+---
+
+1. Create a separate shell script that includes runtime-specific tasks such as clearing cache. That script is executed before your app starts.
+2. Specify an alternative `start` command by adapting the variables:
+
+    ```yaml {location=".platform.app.yaml"}
+    web:
+        commands:
+            start: bash <PATH_TO_FILE>.sh && php <PATH_TO_YOUR_APP>.php
```

Comment:
:hammer_and_wrench: Since `PATH` wasn't being used above and because the path would include the extensions, so it's a little odd to see it separate.

```suggestion
            start: bash <SCRIPT_NAME>.sh && php <APP_FILE_NAME>.php
```